modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/UpdateStyleBookEntryPreviewMVCActionCommandTest.java

```diff
@@ -92,6 +93,60 @@ public void setUp() throws Exception {
 		_themeDisplay.setUser(TestPropsValues.getUser());
 	}
 
+	@Test(expected = NoSuchFileEntryException.class)
+	public void testInvalidStyleBookEntryPreview() throws Exception {
+		MockLiferayPortletActionRequest mockLiferayPortletActionRequest =
```

Comment:
You could move this initialization after `File fileEntry = ...` where its first usage happens

Reply:
This is done in the last commit